R/functions_group_interaction_utility.R

```diff
@@ -59,7 +73,8 @@ defineGroups_interaction <- function(records, actors, seed.randomization) {
 
   # STEP 1: order all events (start and end)
   # if some events occur at the same time, we randomize the order
-  # TODO: if events are close enough (<delta t), we also randomize when unperfect data
+  # TODO: if events are close enough (<delta t), we also randomize when
```

Comment:
Add this as an issue to track rather than in-code comment

Reply:
@marion-hoffman, thoughs?